packages/medusa/src/api/routes/admin/discounts/index.ts

```diff
@@ -91,10 +96,19 @@ export const defaultAdminDiscountsRelations = [
   "rule.conditions",
 ]
 
+export const defaultAdminDiscountConditionFields: (keyof DiscountCondition)[] =
+  ["id", "type", "operator", "discount_rule_id", "created_at", "updated_at"]
+
+export const defaultAdminDiscountConditionRelations = ["discount_rule"]
+
 export type AdminDiscountsRes = {
   discount: Discount
 }
 
+export type AdminDiscountConditionsRes = {
```

Comment:
We also need to export the query param class from `get-condition`

Reply:
So we can use the query type when we implement the endpoint in `medusa-js` and `medusa-react`

Reply:
I'll go ahead and include the endpoint in those two packages now. Thanks for reminding me @kasperkristensen  😅